test/plugin_functional/test_suites/core_plugins/rendering.ts

```diff
@@ -277,6 +277,7 @@ export default function ({ getService }: PluginFunctionalProviderContext) {
         'xpack.index_management.editableIndexSettings (any)',
         'xpack.index_management.enableDataStreamsStorageColumn (any)',
         'xpack.infra.sources.default.fields.message (array)',
+        'xpack.index_management.enableTogglingDataRetention (any)',
```

Comment:
__optional nit:__
```suggestion
        'xpack.index_management.enableTogglingDataRetention (any)', // It's a boolean (any because schema.conditional)
```